ui/src/environments/environment-type.ts

```diff
@@ -37,6 +37,10 @@ export interface Environment {
   // The base URL for the Zendesk help center / user forum.
   // Example value: https://aousupporthelp.zendesk.com/hc/
   zendeskHelpCenterUrl: string;
+  // The Shibboleth URL for linking ERA Commons accounts
+  shibbolethUrl: string;
+
+  rootUrl: string;
```

Comment:
Were you unable to just get this from the window.location instead? I'd rather avoid encoding it here if possible since it's inherent to the application.

Reply:
we want to use the homepage specifically as the redirect callback even if we are on another page.  this seemed cleaner than putting some parsing logic on window.location. what do you think?